src/lib.rs

```diff
@@ -73,7 +73,7 @@ pub enum PackageManager {
 
 /// Obtain the amount of installed packages on the system by checking all installed supported package
 /// managers and adding the amounts
-pub fn total_packages(package_readout: &PackageReadout) -> usize {
+pub fn total_packages(package_readout: &PackageReadout, skip_slow_package_managers: bool) -> usize {
```

Comment:
Can you rename it to `skip_slow` in the `lib.rs` again :), I only meant to change it in the `main.rs` as here is clear what is skipped